tools/matchms/matchms_filtering.xml

```diff
@@ -69,16 +69,22 @@
         <conditional name="normalise_intensities">
             <param name="is_true" label="Normalize intensities" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="false"
                    help="Normalize intensities of peaks (and losses) to unit height."/>
+            <when value="TRUE"></when>
```

Comment:
Why do you use a conditional here, if there is no conditional?  It could be a simple boolean. 
Also please see https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#booleans
in conditionals we recommend select-type and not booleans.